### PR TITLE
alt+drag should duplicate all selected elems

### PR DESCRIPTION
**Pull request by dwelle**
additional fix for #255

excalidraw_alt_drag_duplicate_many

### CODE REVIEW

**Comment by dwelle** [comment]
That first commit is actually already in master. Dunno what will happen when squashing this PR.

**Comment by dwelle** [comment]
Actually, let's not merge this yet. Duplicating text elements doesn't work.

**Comment by jeremyscatigna** [comment]
Didn't test text elements, maybe I could continue working on the feature to make it more complete if needed

**Comment by dwelle** [comment]
Looks ok now.

**Comment by vjeux** [comment]
Looks awesome thanks!

**Comment by vjeux** [comment]
It's not working for text nodes :(

File: src/index.tsx [hunk 1]
```diff
@@ -680,21 +680,15 @@ class App extends React.Component<{}, AppState> {
...
-                      hitElement.strokeWidth,
-                      hitElement.roughness,
-                      hitElement.opacity,
-                      hitElement.width,
-                      hitElement.height
+                    elements.push(
+                      ...elements.reduce((duplicates, element) => {
+                        if (element.isSelected) {
+                          duplicates.push({ ...element });
+                          element.isSelected = false;
```
Comment (vjeux) [inline src/index.tsx]: missing generateDraw() command

Comment (vjeux) [reply src/index.tsx]: Also regenerating a new seed

Comment (dwelle) [reply src/index.tsx]: It should be copied, too. Works for me :thinking:

Comment (dwelle) [reply src/index.tsx]: excalidraw_alt_drag_duplicate_texts

Comment (dwelle) [reply src/index.tsx]: > Also regenerating a new seed

ah.. and `id`.. Man. This has been a debacle :). Will fix after dinner. Can you still repro the text issue?

**Pull request merged**